src/adagrams.js

```diff
@@ -1,8 +1,160 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+class Adagrams {
+  static drawLetters() {
+
+    let letters = [
+      'A','A','A','A','A','A','A','A','A',
+      'B','B',
+      'C','C',
+      'D','D','D','D',
+      'E','E','E','E','E','E','E','E','E','E','E','E',
+      'F','F',
+      'G','G','G',
+      'H','H',
+      'I','I','I','I','I','I','I','I','I',
+      'J','J',
+      'K',
+      'L','L','L','L',
+      'M','M',
+      'N','N','N','N','N','N',
+      'O','O','O','O','O','O','O','O',
+      'P','P',
+      'Q',
+      'R','R','R','R','R','R',
+      'S','S','S','S',
+      'T','T','T','T','T','T',
+      'U','U','U','U',
+      'V','V',
+      'W','W',
+      'X',
+      'Y','Y',
+      'Z',
+    ];
+
+    let number = letters.length;
+    
+    function getRandomInt(number) {
+      return Math.floor(Math.random() * Math.floor(number));
+    }
+
+    let randomIndexes = [];
+
+    for (let i = 0; i < 10; i += 1){
+      let random = getRandomInt(number);
+      if (randomIndexes.includes(random)){
+        getRandomInt(number);
+        i -= 1
```

Comment:
Hm, you have a case in which your logic makes the `i` index variable need to do special things... In this case, converting this to a `while` loop may be more readable (consider `while (randomIndexes.length < 11)` instead of the `for`)